src/components/Tool/Appeal/InitialPage/Appeal.tsx

```diff
@@ -105,7 +105,7 @@ const Appeal = ({
               className={classes.nameLink}
             >
               <NextLink
-                href={`/accountLists/${accountListId}/tools/appeals/${id}`}
+                href={`/accountLists/${accountListId}/tools/appeals/appeal/${id}`}
```

Comment:
Do we need /appeal/ here? It seems a little repetitive.

Reply:
Yes, due to the flashing appeal initial page issue with Next.js